### ISSUE TITLE
uptime monitor email registration intent

### ISSUE THREAD

**Issue by ineagu** [Codeinwp/themeisle-companion#207]
@selul when a user activate uptime module, we should display a popup asking : " Where should we send the notification if mysite.com goes down? " and add an input pre-filled with current user email + a. confirm button, only then send the request.

Even a safer approach would be to then send a confirmation email to that address and only when a user confirms the email address, confirm the monitor.

Imagine that there can be users entering wrong emails or random emails that we'll keep emailing and get our IP reputation down or spam reports.

**Comment by ineagu** [comment]
@selul I would also show this module as 2nd after the share one

@abaicus let's mention this on landing page/demo as well

**Comment by selul** [comment]
@abaicus can you help me in adding the popup confirmation? Some thoughts that I see in order to avoid too much hassle and add the registration intent: 
* add a module propriety, called confirm intent. The propriety will be an array that will hold two keys, title and description, which will be used in the popup. 
* after they will see the popup, they will have one button for confirmation, which when clicked will focus directly on the module controls. 
Basically the workflow will be something like this: 

1. Activate module.
2. User see the popup and click on button for intent confirmation. 
3. Focus on the module related fields. 
4. User fill in the email, click save.
5. The monitor is scheduled. 

On module deactivate, we will keep the current logic. 


The confirmation email workflow can be done later on.
